signer/signer.go

```diff
@@ -0,0 +1,262 @@
+package signer
+
+import (
+	"crypto/x509"
+	"fmt"
+	"time"
+
+	nx509 "github.com/notaryproject/notation-core-go/x509"
+)
+
+type SignatureMediaType string
```

Comment:
Missing documentation for `SignatureMediaType` and other exported structs.

Reply:
Added